devel/libblocksruntime/Portfile

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libblocksruntime
+version             0.4.1
+categories          devel
+platforms           darwin
+license             MIT
+maintainers         gmail.org:adsun701
+description         compiler-rt Blocks runtime library for Clang
+long_description    libblocksruntime is a target-independent implementation of Apple "Blocks" runtime interfaces.
+homepage            https://compiler-rt.llvm.org
+master_sites        http://ftp.debian.org/debian/pool/main/libb/libblocksruntime/
+distname            ${name}_${version}.orig
+worksrcdir          ${name}-${version}
+checksums           rmd160  43196c58a5c85b48ec94a583bd67dc2120467977 \
+                    sha256  7807e18d7d6cabd90f77c8b8a8ebd472516fa4ed6a02df82e0c33b1c5e112abc \
+                    size    312180
+depends_build       port:clang_select
```

Comment:
In what way does it require clang_select at build time? clang_select, like all *_select ports, is for a user's convenience only; ports should not require them.

Reply:
@Adsun701 Please submit a PR also requesting the removal of the clang_select dependency.